src/datasources/alerts-api/tenderly-api.service.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { faker } from '@faker-js/faker';
+import { FakeConfigurationService } from '@/config/__tests__/fake.configuration.service';
+import { TenderlyApi } from '@/datasources/alerts-api/tenderly-api.service';
+import { HttpErrorFactory } from '@/datasources/errors/http-error-factory';
+import { AxiosNetworkService } from '@/datasources/network/axios.network.service';
+import { Contract, ContractId } from '@/domain/alerts/entities/alerts.entity';
+import { DataSourceError } from '@/domain/errors/data-source.error';
+
+const networkService = {
+  post: jest.fn(),
+  delete: jest.fn(),
+} as unknown as AxiosNetworkService;
```

Comment:
We should use the interface type here instead of the actual implementation. If we start using a different HTTP client this would mean a new implementation of `INetworkService`.

Using the reference to `INetworkService` instead would make this test compatible with the change that I described.

Reply:
I used a few other tests as reference here. I've adjusted this in 36844eeae590fc1a6dd9d1ec0e87d8645e760b2d and will update the other tests I used as a reference in another PR.